Automattic/vip-go-mu-plugins
# Add Jetpack 9.2 subtree

## Description

Add new subtree for Jetpack 9.2. The initial version corresponds to tag 9.2-beta

## Checklist


## Steps to Test

1. Check out PR.
1. Add `define( 'VIP_JETPACK_PINNED_VERSION', '9.2' );`
1. Check that the Jetpack dashboard points to version `9.2-beta`
